bookkeeper-server/src/test/java/org/apache/bookkeeper/bookie/BookieInitializationTest.java

```diff
@@ -270,9 +270,7 @@ public void testDuplicateBookieServerStartup() throws Exception {
             fail("Should throw BindException, as the bk server is already running!");
         } catch (ChannelException ce) {
             Assert.assertTrue("Should be caused by a bind exception",
-                              ce.getCause() instanceof BindException);
-            Assert.assertTrue("BKServer allowed duplicate startups!",
```

Comment:
In italian locale  the error is like "indirizzo già in uso" and the test fails. This is annoying on every machine I have and my colleagues